operations/spire/Update_Spire_Intermediate_CA_Certificate.md

```diff
@@ -0,0 +1,49 @@
+# Update Spire intermediate CA certificate
+
+Prior to CSM 1.2.5 there is no mechanism to automatically update the spire
+intermediate CA certificate before it expires. This certificate expires after
+one year. Administrators will want to keep track of when this certificate
+expires and manually update the certificate before it expires.
+
+## How to obtain the expiration date for the Spire intermediate CA
+
+To obtain the expiration date of the Spire intermediate CA certificate, run the
+following command on a node that has access to kubectl (Such as ncn-m001)
+
+```bash
+kubectl get secret -n spire spire.spire.ca-tls -o json | jq -r '.data."tls.crt" | @base64d' | openssl x509 -noout -enddate
+```
+
+## How to replace the Spire intermediate CA certificate with a new one
+
+certificate you need to delete the secret that stores the certificate and then
```

Comment:
I think this line above is just a duplicate? copy/paste error?

Reply:
Looks like it. Fixed.